server/tests/administration/test_create_organization.py

```diff
@@ -71,6 +72,11 @@ async def test_bad_method(
     assert response.status_code == 405, response.content
 
 
+class CreateOrganizationParams(TypedDict):
+    active_user_limit: NotRequired[int | None]
+    user_profile_outsider_allowed: NotRequired[bool]
+
+
```

Comment:
I think `reportMissingTypeArgument` could also be disabled for the tests, although having it enabled is not a big deal either.